tests/endtoend/test_eventgrid_functions.py

```diff
@@ -69,17 +86,82 @@ def test_eventgrid_trigger(self):
                 response = r.json()
 
                 self.assertEqual(
-                    response,
-                    {
-                        'id': data[0]['id'],
-                        'data': data[0]['data'],
-                        'topic': data[0]['topic'],
-                        'subject': data[0]['subject'],
-                        'event_type': data[0]['eventType'],
-                    }
+                    response, {'id': data[0]['id'], 'data': data[0]['data'],
```

Comment:
Syntactic sugar when comparing two dicts, checking if response is a subset of data[0]:
```python
self.assertLessEqual(response.items(), data[0].items())
```